### DISCUSSION TITLE
Custom retention policy for spans in Kafka

### DISCUSSION THREAD

**Discussion by yury-kozlov** [jaegertracing/jaeger#3137] [Q&A] [answered]
We are setting up Jaeger (using jaeger-operator) with Kafka and Elasticsearch on premises.
Is there an option to define custom retention policy for spans stored in Kafka (e.g. one hour)?
Or I can only configure it in Kafka directly ?

**Comment by yurishkuro** [comment, marked as answer]
To my knowledge this is something you control via Kafka itself, not from the producer side.
